models/orden_de_fabricacion.ts

```diff
@@ -0,0 +1,12 @@
+import { Modelo } from "./modelo.ts";
+
+//Contiene uno o más paquetes de modelos y el número de unidades por modelo que deben fabricarse
+export class OrdenDeFabricacion{
+    
+    //Conjunto de modelos con los números de unidades
+    modelos: Array<[Modelo, number]>;
+
+    constructor(modelos:Array<[Modelo, number]>){
```

Comment:
```suggestion
    modelos: Array<ModeloFabricacion>;

    constructor(modelos:Array<ModeloFabricacion>){
```
@vdeq79 aquí creo que sería más conveniente hacer un objeto valor intermedio, ya que de ese modo se podrá identificar que number (unidades) es de qué Modelo y además así será más sencillo de modelar en un futuro en la base de datos en la propia tabla de relación N:N.

Bastaría con crear ese objeto valor con dos propiedades, la primera el modelo y la segunda numérica para las unidades.

Reply:
Quieres decir que un modelo puede estar en distintos órdenes de fabricación? Es que no acabo de entender muy bien el concepto de orden de fabricación.

Reply:
Sí, un mismo modelo podría fabricarse varias veces, por ejemplo en el caso de una fábrica de camisas:
- Una operación puede ser: Coser botones (una misma operación también puede estar en varios modelos)
- Un modelo puede ser: Camisa de rayas (que tendrá una serie de operaciones como la anterior)
- Una orden de fabricación es simplemente la orden que se le pasa a los operarios de que hay que fabricar, por ejemplo:
> - 5 camisas de rayas (con x operaciones)
> - 20 camisas de cuadros (con y operaciones)